### DISCUSSION TITLE
Welcome to rtl_433 Discussions!

### DISCUSSION THREAD

**Discussion by merbanan** [merbanan/rtl_433#2067] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Saamir20** [comment]
I am new to Github, and new to rtl_433. Remembering a few things about git, I would like to 1) clone the repository, 2) I assume I need to create a branch, 3) edit my changes and test them, then 4) create a pull request. But,

There's no documentation (that I can find) to contribute to this body of work. I cannot find any "clone" button. I would think that someone, who was once a newbie like myself, would create the documentation that would walk me through this, and make that documentation easy to find. I'm talking 30 minutes of work. Wow.

Can somebody show me where Step 1 is?

**Reply by zuckschwerdt** [reply]
This should get you started:
1) https://docs.github.com/articles/fork-a-repo
2) https://docs.github.com/articles/cloning-a-repository 
3) https://docs.github.com/articles/about-branches 
4) https://docs.github.com/articles/creating-a-pull-request-from-a-fork

**Comment by merbanan** [comment]
Hi, this level of support is not available from the rtl_433 project. I would suggest you look through some github tutorials on youtube and read through all the available documentation that we have. After that you should be set on how to use rtl_433 and how to contribute to any github project.

**Comment by norby7478** [comment]
Hi everyone. I don't know, but after all the reading in the forum I am still stuck.

![image](https://github.com/merbanan/rtl_433/assets/118662816/91ac2970-eb76-4653-81e1-a5dd3989fde5)

I am always getting "it is not a valid repository". And yes, I have tried the way with "repository" as well as "integration"
And not only this one (which at present this would be the most important for me), but so many others I tried and the same.

Did somebody  else have this issue?

Thanks.

**Reply by HolgerAusB** [reply]
??? what do you mean? Where are you trying to 'add' this repo and how?

**Comment by norby7478** [comment]
Dear Holger,

If I choose the "Add-ons" button

![image](https://github.com/merbanan/rtl_433/assets/118662816/cf04bedd-8002-47e1-b752-073d125a3eab)

then 
![image](https://github.com/merbanan/rtl_433/assets/118662816/d02dff92-fdec-4692-b1e2-3bd4b3cbdf52)

I choose 
![image](https://github.com/merbanan/rtl_433/assets/118662816/283cafeb-4662-4cdc-af8f-eb65e97f00fb)
then top right the 
![image](https://github.com/merbanan/rtl_433/assets/118662816/5841b6ef-f56e-4422-b227-9501f372af24) after 
![image](https://github.com/merbanan/rtl_433/assets/118662816/9707cc78-0a78-47a4-82f6-1f7a78dcb7c2)
 copy the Url 
![image](https://github.com/merbanan/rtl_433/assets/118662816/081c5d2b-5663-4fc5-b841-6d1f4f86b0bd)

Click "Add" I get 
![image](https://github.com/merbanan/rtl_433/assets/118662816/3f8e77a5-101e-47ec-9f77-4738ca27d872)

Then there have been comments, that this is not an add on, so I went instead trying integration:

![image](https://github.com/merbanan/rtl_433/assets/118662816/cdad8fbf-820b-40fb-bd75-5e9417223799)

![image](https://github.com/merbanan/rtl_433/assets/118662816/eab4cb6c-efa2-4bc0-a30f-a9cc24b13a8f)

![image](https://github.com/merbanan/rtl_433/assets/118662816/1b7707d4-7460-48e8-895d-d899bca63c37)

![image](https://github.com/merbanan/rtl_433/assets/118662816/1cc9ff51-817c-4037-8808-941f5062d344)

And that is the result:

![image](https://github.com/merbanan/rtl_433/assets/118662816/e311f4bb-106c-4a0f-9717-2a9a35ec194f)

I am very sorry for this long message, but I hope a solution or a hint where I am doing things wrong would also help others.

Thanks a lot and a good evening,

N

**Reply by zuckschwerdt** [reply]
rtl_433 is not a HA plugin. You might be looking for https://github.com/pbkhrv/rtl_433-hass-addons

**Comment by norby7478** [comment]
Sorry for the late reply. Your help is really appreciated.
But I think I am not up to the mark with this HACS repositories. Running the latest version of Home assistant.

Result:
![image](https://github.com/merbanan/rtl_433/assets/118662816/8a435c4a-246f-495f-ab74-1400c4b791ab)
![image](https://github.com/merbanan/rtl_433/assets/118662816/8a5e4685-4d38-4ea4-9804-3e76266d0cd3)

I have no idea what I am doing wrong. The steps in the readme file were followed, and it should be working, because I don't think I am the only one who wants to install this.

Warm regards,

N

**Comment by pswiatki** [comment]
Hello there! I am a software engineer, formerly with the automotive industry for many years. I worked in the infotainment segment (DAB receivers for vehicles, mostly). My interest in `rtl_433` is mainly to decode radio signals from devices around the house, but if there is an opportunity to contribute, I will. For now mostly in the testing of decoders, verification of their output, etc. 
I already started, see #3062  or (soon) #3066 and have some questions I would like to ask here. Perhaps trivial, but we'll see.
